packages/cdk/apps/context/app.ts

```diff
@@ -33,5 +33,6 @@ new ContextStack(app, `${PRODUCT_NAME}-Context-${contextParameters.projectName}-
     [USER_ID_TAG_KEY]: contextParameters.userId,
     [USER_EMAIL_TAG_KEY]: contextParameters.userEmail,
     [AGC_VERSION_KEY]: contextParameters.agcVersion,
+    ...contextParameters.customTags,
```

Comment:
spreading `customTags` last at the end of `tags` is dangerous because any custom tag key will override a previous key, this could lead to some unintended consequences if a "reserved" key is used as a custom key

for instance 

```
const customTags = {
    "AGC_VERSION_KEY": "hotdog"
  }

const tags = {
    "AGC_VERSION_KEY": 1,
    ...customTags
  }

console.log(tags["AGC_VERSION_KEY"]) // "hotdog"
```